training/model_evaluation/environment/spec.yaml

```diff
@@ -0,0 +1,17 @@
+$schema: https://azuremlschemas.azureedge.net/latest/environment.schema.json
+description: >-
+  An environment for mlflow based model evaluation tasks. 
```

Comment:
Environment is no longer for mlflow-only..

An environment for Model Evaluation tasks. (or)